docs/flow/monitoring/component_metrics.md

```diff
@@ -6,3 +6,32 @@ weight: 200
 ---
 
 # Component metrics
+
+Grafana Agent Flow [components][] may optionally expose Prometheus metrics
+which can be used to investigate the behavior of that component. These
+component-specific metrics are only generated when an instance of that
+component is running.
+
+> Component-specific metrics are different than any metrics being processed by
+> the component. Component-specific metrics are used to expose the state of a
+> component for observability, alerting, and debugging.
+
+Metrics for component-specific metrics are exposed at the `/metrics` HTTP
```

Comment:
Should this just be "Component-specific metrics are exposed..."?  Are they actually metrics for metrics?

Reply:
> Should this just be "Component-specific metrics are exposed..."?

Oops, yes :)